server/datastore/mysql/hosts.go

```
@@ -0,0 +1,242 @@
+package mysql
+
+import (
+	"database/sql"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/jmoiron/sqlx"
+	"github.com/kolide/kolide-ose/server/errors"
+	"github.com/kolide/kolide-ose/server/kolide"
+)
+
+func (d *Datastore) NewHost(host *kolide.Host) (*kolide.Host, error) {
+	sqlStatement := `
+	INSERT INTO hosts (
+		detail_update_time,
+		node_key,
+		host_name,
+		uuid,
+		platform,
+		osquery_version,
+		os_version,
+		uptime,
+		physical_memory,
+		primary_mac,
+		primary_ip
+	)
+	VALUES( ?,?,?,?,?,?,?,?,?,?,?)
+	`
+	result, err := d.db.Exec(sqlStatement, host.DetailUpdateTime,
+		host.NodeKey, host.HostName, host.UUID, host.Platform, host.OsqueryVersion,
+		host.OSVersion, host.Uptime, host.PhysicalMemory, host.PrimaryMAC, host.PrimaryIP)
+	if err != nil {
+		return nil, errors.DatabaseError(err)
+	}
+	id, _ := result.LastInsertId()
```

ignoring error here? http://go-database-sql.org/ implies we should check.